www/docs/src/content/docs/start-here/getting-started.mdx

```diff
@@ -93,11 +93,40 @@ export default defineConfig({
 Astro StudioCMS requires SSR mode to work correctly. Make sure to set `output: 'server'` and an adapter that supports SSR in your `astro.config.mjs` file.
 :::
 
-### Running your Astro project
+## Configure your `package.json` scripts
+
+Setup your `package.json` scripts to include the `--remote` flag for build, and optionally for dev. (You can also run the dev command as shown in the ["Running your StudioCMS Project"](#running-your-studiocms-project) section)
+
+```json title="package.json"
+{
+    "name": "my-studiocms-project",
+    "scripts": {
+        "dev": "astro dev --remote",
+        "build": "astro check & astro build --remote",
+        "astro": "astro"
+    }
+}
+```
+
+## Running your StudioCMS project
+
+Thanks to the power of Astro running StudioCMS is as easy as running the dev command for local preview, or building and deploying to your server, for the basics of how to use it locally without building here is what you need to do.
+
```

Comment:
This doesn't provide any useful info, so I'd probably remove it unless there's something specific to call out here.
```suggestion
```

Reply:
I admit it's mainly to help with spacing of the headings at the moment... 😅

I was thinking once [PR#11385](https://github.com/withastro/astro/pull/11385) switch this small section to mention specifically the AstroDB environment variable options for running with a Self-hosted LibSQL vs Astro Studio

Reply:
h2 -> h3 is fine, and if we want to add something later, that's fine. still, no reason to block this whether it's removed/replaced/remains, so I'll approve